### DISCUSSION TITLE
Python updates everytime I open chaiNNer and takes hours to update.

### DISCUSSION THREAD

**Discussion by Benbarian** [chaiNNer-org/chaiNNer#2290] [Q&A]
I never get to use chaiNNer because every time I open it, python updates for hours and hours and I eventually go fin da differnt online upscaler. Can I force it to work without having to update each time?

**Comment by RunDevelopment** [comment]
As long as you didn't update chaiNNer itself, this should not happen. After chaiNNer installs its python dependencies the first time around, it has no reason to update/reinstall them, so something is going wrong.

Could you send us your logs, so we can figure out the issue? In the menu, go to Help > Open Logs Folder and send us `main.log` and `renderer.log`. 

If you don't want to open chaiNNer, the folder is located at:
- If you installed chaiNNer on Windows with the installer: `%appdata%\chaiNNer\logs`
- If you installed a portable version of chaiNNer, the logs are in the `logs` folder of your install directory.

**Reply by Benbarian** [reply]
[main.log](https://github.com/chaiNNer-org/chaiNNer/files/13205488/main.log)
[renderer.log](https://github.com/chaiNNer-org/chaiNNer/files/13205489/renderer.log)

**Comment by Benbarian** [comment]
Thank you so much for your reply. Attached are my logs. It feels like every
time I open the app I have to download from repositories. I think i ialso
isntalled a latest vrsion of the app today.

On Mon, 30 Oct 2023 at 14:29, Michael Schmidt ***@***.***>
wrote:

> As long as you didn't update chaiNNer itself, this should not happen.
> After chaiNNer installs its python dependencies the first time around, it
> has no reason to update/reinstall them, so something is going wrong.
>
> Could you send us your logs, so we can figure out the issue? In the menu,
> go to Help > Open Logs Folder and send us main.log and renderer.log.
>
> If you don't want to open chaiNNer, the folder is located at:
>
>    - If you installed chaiNNer on Windows with the installer:
>    %appdata%\chaiNNer\logs
>    - If you installed a portable version of chaiNNer, the logs are in the
>    logs folder of your install directory.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chaiNNer-org/chaiNNer/discussions/2290#discussioncomment-7423427>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACUVUSIQVZCPQ32PSMKLJDYB6MRTAVCNFSM6AAAAAA6VWJ5DSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMRTGQZDO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
B.Nel

---
 “It is no measure of health to be well adjusted to a profoundly sick
society.”
J. Krishnamurti

**Reply by joeyballentine** [reply]
@Benbarian can you attach those through GitHub? I'm assuming you did it through email since they aren't appearing

**Comment by Benbarian** [comment]
Will do, apologies

On Mon, 30 Oct 2023 at 15:09, Joey Ballentine ***@***.***>
wrote:

> @Benbarian <https://github.com/Benbarian> can you attach those through
> GitHub? I'm assuming you did it through email since they aren't appearing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chaiNNer-org/chaiNNer/discussions/2290#discussioncomment-7423865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACUVUW2MKW2LED2NRPDCCDYB6RKBAVCNFSM6AAAAAA6VWJ5DSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMRTHA3DK>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
B.Nel

---
 “It is no measure of health to be well adjusted to a profoundly sick
society.”
J. Krishnamurti
